packages/predicate/src/predicate.ts

```diff
@@ -111,18 +112,31 @@ export class Predicate<ARGS extends InputValue[]> extends Account implements Abs
    * @returns The Predicate instance with updated predicate data.
    */
   setData<T extends ARGS>(...args: T) {
+    this.predicateArgs = args;
+
+    return this;
+  }
+
+  #getPredicateData(txLength: number): Uint8Array {
```

Comment:
Following the previous suggestion.

```suggestion
  #getPredicateData(policiesLength: number): Uint8Array {
```